src/main/java/ua/javaPro/berezhnoy/hillelHomeWork6/phonebook/Phonebook.java

```diff
@@ -0,0 +1,36 @@
+package ua.javaPro.berezhnoy.hillelHomeWork6.phonebook;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Phonebook {
+    static HashMap<String, ArrayList<String>> name = new HashMap<>();
```

Comment:
```suggestion
    static Map<String, List<String>> names = new HashMap<>();
```
краще працювати з інтерфейсами. також змінну краще в множині писати.